Helper/Data.php

```diff
@@ -150,6 +149,11 @@ class Data extends AbstractHelper
      */
     const XML_PATH_TAG_DATE_PUBLISHED = 'nosto/flags/tag_date_published';
 
+    /**
+     * Path to the configuration object that stores customer reference
+     */
+    const XML_PATH_CUSTOMER_REFERENCE = 'nosto/flags/customer_reference';
```

Comment:
This flag is not really for using or generating customer reference, right? It's about sending orders to Nosto without Nosto customer id? Please rename to something like `XML_TRACK_MULTI_CHANNEL_ORDERS = 'nosto/flags/track_multi_channel_orders'`